controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -485,6 +502,20 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           }
         }
 
+        if (req.gitInfo && opts.githubApp) {
+          const githubRepo = new GitHubRepository(opts.db, opts.githubApp);
+          githubRepo.createCommitCheck({
```

Comment:
Missing `.catch` otherwise it leads to `unhandledPromiseRejection`

Reply:
Added await